Source/Initialization/PlasmaInjector.cpp

```diff
@@ -88,6 +88,30 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
     ymax = std::numeric_limits<amrex::Real>::max();
     zmax = std::numeric_limits<amrex::Real>::max();
 
+    // NOTE: When periodic boundaries are used, default injection range is set to mother grid dimensions.
+    const Geometry& geom = WarpX::GetInstance().Geom(0);
+    if( geom.isPeriodic(0)==1 ) {
```

Comment:
Since this returns a `bool` we could make this more readability by
```suggestion
    if( geom.isPeriodic(0) ==true ) {
```
or even
```suggestion
    if( geom.isPeriodic(0) ) {
```